geopyspark/geotrellis/__init__.py

```diff
@@ -394,6 +394,33 @@ def __new__(cls, tile_size=None, tile_cols=None, tile_rows=None):
 RasterizerOptions.__new__.__defaults__ = (True, 'PixelIsPoint')
 
 
+SpatialPoint = namedtuple("SpatialPoint", 'point values')
```

Comment:
@echeipesh I want to get your opinion on the `SpatialPoint` and `SpaceTimePoint` types. Currently, they are simply wrappers around `(Point, [int or float])` pairs and don't serve much of a purpose beyond that. Unless we plan on expanding these types and their uses, I think it might be easier to just return a `{point: [int or float]}` instead. Because when I was working with these types, I found they'd be better suited as `dict`s in their current state.